terra/tools/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "terra-contract-tools",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build-contracts": "( cd .. && docker run --rm -v \"$(pwd)\":/code --mount type=volume,source=\"$(basename \"$(pwd)\")_cache\",target=/code/target --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry cosmwasm/workspace-optimizer:0.10.4 )",
```

Comment:
This step should be part of the Tilt flow to enable hot reloading like for the Solana/ETH contracts (I can build this if you want)